src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -182,48 +162,26 @@ public XmlLoggingConfiguration([NotNull] XmlReader reader, [CanBeNull] string fi
         /// <param name="fileName">Name of the file that contains the element (to be used as a base for including other files). <c>null</c> is allowed.</param>
         /// <param name="ignoreErrors">Ignore any errors during configuration.</param>
         /// <param name="logFactory">The <see cref="LogFactory" /> to which to apply any applicable configuration values.</param>
-        public XmlLoggingConfiguration([NotNull] XmlReader reader, string fileName, bool ignoreErrors, LogFactory logFactory)
-            : base(logFactory)
-        {
-            Initialize(reader, fileName, ignoreErrors);
-        }
+        [Obsolete("Constructor with parameter ignoreErrors is no longer supported. Marked obsolete in NLog 5.0")]
```

Comment:
maybe add that ignoreErrors is now not used anymore? (2 times). I the XMLdoc and/or obsolete message.

Reply:
Changed `ignoreErrors is no longer supported` to `ignoreErrors is no longer supported and has no effect`. Is it better?